<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventForm</class>
 <widget class="QDialog" name="AddEventForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Event</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelTime">
      <property name="text">
       <string>Time: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelAct">
      <property name="text">
       <string>Action: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="actTypeBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
